django/contrib/postgres/fields/ranges.py

```diff
@@ -112,6 +117,34 @@ def formfield(self, **kwargs):
         return super().formfield(**kwargs)
 
 
+class ContinuousRangeField(RangeField):
+    """
+    This class specializes RangeField allowing user to choose a default bounds value
+    for the field. This assumption is conservative in the sense that we enforce the
+    chosen 'default_bounds' exclusively for inputs of the types (list, tuple).
+    We're not replacing the bounds attribute by 'default_bounds' for objects of the type
+    `psycopg2.extras.Range`.
+    """
+
+    def __init__(self, *args, default_bounds=CANONICAL_RANGE_BOUNDS, **kwargs):
+        self.default_bounds = default_bounds
```

Comment:
Should we preemptively validate `default_bounds` to be one of the four possible values ?  Misuse may cause errors down the stack.

Reply:
Yes, we should raise a `ValueError` for incorrect values.

Reply:
Validation was added.